internal/pkg/vm/internal/vmcontext/invocation_context.go

```diff
@@ -307,11 +308,16 @@ func (ctx *invocationContext) Send(toAddr address.Address, methodNum abi.MethodN
 		}
 	}()
 
+	// replace nil params with empty value
+	if params == nil {
```

Comment:
@icorderi Added this to handle a specs actor method that returned nil.  Maybe we should enforce this in actor code instead.

Reply:
This should definitely be enforced in actor code, the tests are just insufficient. I don't love this fallback, but ok if you must.

Please propose a fix for the actor in question

Reply:
This is the culprit: https://github.com/filecoin-project/specs-actors/pull/214.  I'll patch go-filecoin up once it lands.